### PR TITLE
Hotfix/theme conflict bodyStyle variable (body.style attribute) #572

### PR DESCRIPTION
**Pull request by idevit**
This pull request fixes bodyStyle variable (body.style attribute) #572

https://github.com/RocketChat/Rocket.Chat.Livechat/issues/572

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1,15668 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (MartinSchoeler) [inline yarn.lock:1]: Could you please undo this deletion? Thanks!

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
Hello! Could you do a rebase to the current develop? It seems that we have some conflicting changes in here. Thanks!

**Comment by idevit** [comment]
why is the lock not in the gitignore?
sorry for responding late were looking into it

**Comment by idevit** [comment]
it should be ok now?

**Comment by murtaza98** [comment]
@idevit Thanks for the contribution!! I found some issues with this PR while testing. The thing is, scrolling stopped working on mobile devices, as soon a widget was opened and then closed. I believe this was a result of adding these 2 extra CSS attributes to the body -> `position: fixed; top: ${ scrollPosition }px;` 

So, I've created a PR https://github.com/RocketChat/Rocket.Chat.Livechat/pull/660 to fix this and also simplified the logic for handling styles by grouping those styles into a single class. Feel free to test the change and share your findings. Thanks!!

**Pull request closed without merging**